### DISCUSSION TITLE
Spawning with child objects

### DISCUSSION THREAD

**Discussion by stilnat** [FirstGearGames/FishNet#392] [Documentation]
I believe [this page](https://fish-networking.gitbook.io/docs/manual/guides/spawning) could contain a bit more doc regarding spawning with child game objects. 
In particular, It'd be nice to have some doc regarding what happens when spawning with enabled (or disabled) child game objects on the prefab I want to spawn.
Discussing with Max Kratt on Discord, and experimenting, I realized that my child game objects need to be enabled on the prefab when spawning, otherwise they don't spawn on client. 
That is not written anywhere I know of.

**Comment by maxkratt** [comment]
This page kinda describes that: https://fish-networking.gitbook.io/docs/manual/guides/spawning/nested-networkobjects

**Reply by stilnat** [reply]
right thanks !
